src/throttler-storage-redis.service.ts

```diff
@@ -4,11 +4,7 @@ import { ThrottlerStorageRedis } from './throttler-storage-redis.interface';
 
 @Injectable()
 export class ThrottlerStorageRedisService implements ThrottlerStorageRedis {
-  redis: Redis.Redis;
-
-  constructor(options?: Redis.RedisOptions) {
-    this.redis = new Redis(options);
-  }
+  constructor(private redis: Redis.Redis) {}
```

Comment:
I suppose this would work with a static create method as well, but could it be done to where either a redis instance _or_ options can be passed in? I think it would make for a better DX.